source/extensions/filters/common/ext_authz/ext_authz_http_impl.h

```diff
@@ -45,13 +44,15 @@ class RawHttpClientImpl : public Client,
 
 private:
   ResponsePtr messageToResponse(Http::MessagePtr message);
-  const std::string cluster_name_;
-  const std::string path_prefix_;
-  const Http::LowerCaseStrUnorderedSet& allowed_authorization_headers_;
-  const Http::LowerCaseStrUnorderedSet& allowed_request_headers_;
-  const HeaderKeyValueVector& authorization_headers_to_add_;
-  absl::optional<std::chrono::milliseconds> timeout_;
   Upstream::ClusterManager& cm_;
+  const std::string& cluster_name_;
```

Comment:
Instead of passing all of these variables and storing references can we just store a reference to the config?

Reply:
Yes, I think will look a not cleaner. Thanks!

Reply:
Actually, I will need to move the entire FilterConfig into `filters/common/ext_authz`, but I still think that it will look cleaner.